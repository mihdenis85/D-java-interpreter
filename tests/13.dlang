var myTuple := { x := 5, y, z := 10 }; // 'y' is a pair without a value
print myTuple.x, myTuple.2, myTuple.z; // Should print 5, y, 10