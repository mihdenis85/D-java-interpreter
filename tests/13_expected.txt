5, y, 10